Add initial Online conference prospectus. (#1235)
writethedocs/www: docs/sponsorship/index.rst

Community Sponsorship Options
=============================

> We offer a number of ways to sponsor Write the Docs events.
> We have a number of services we provide our community,
> and each of these is able to be sponsored at a different level:

Reviewer: ```suggestion
We support our community in a variety of ways, and you can choose to focus 
your sponsorship on any one of them, at a level that suits you:
```

> * Our :doc:`/conf/index` each have their own sponsorship prospectus

Reviewer: ```suggestion
* Our conferences each have their own sponsorship prospectus
```

    - Our upcoming :doc:`Portland online conference </conf/portland/2020/sponsors/online-prospectus>`
* Our :doc:`/sponsorship/newsletter` with over {{ newsletter_subs }} subscribers
* Our :doc:`/sponsorship/website` offers both logo sponsorships and advertising 

In addition to these existing programs,
we're always interested in working together with organizations in our community to build new approaches to sponsorship. 

> We believe we have one of the most skilled and foreward looking communities in tech,

Reviewer: ```suggestion
We believe we have one of the most skilled and forward-looking communities in tech,
```
